src/components/InventoryPopover.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <!-- TODO Make values dynamic and internationalise text  -->
+  <ion-select :interface-options="customPopoverOptions" interface="popover" :value="getJobStatus(id) ? getJobStatus(id) : 'SERVICE_DRAFT'" @ionChange="updateJob(id)" >
+    <ion-select-option value="HOURLY">Hourly</ion-select-option>
+    <ion-select-option value="EVERY_6_HOURS">Every 6 hours</ion-select-option>
+    <ion-select-option value="NIGHTLY">Nightly</ion-select-option>
+    <ion-select-option value="SERVICE_DRAFT">Disabled</ion-select-option>
+  </ion-select>
+</template>
+<script lang="ts">
+import { IonSelect, IonSelectOption } from '@ionic/vue';
+import { defineComponent } from 'vue';
+import { useStore } from "@/store";
+import { mapGetters } from "vuex";
+
+export default defineComponent({
+  name: 'InventoryPopover',
+  props: { 
+    id: String
+  },
+  components: {
+    IonSelect,
+    IonSelectOption
+  },
+  computed: {
+    ...mapGetters({
+      getJobStatus: 'job/getJobStatus',
+      getJob: 'job/getJob'
+    })
+  },
+  methods: {
+    async updateJob(id: string) {
+      const job = this.getJob(id);
+      const payload = {
+        ...job,
+        'systemJobEnumId': id,
+        'statusId' : "SERVICE_PENDING"
+      } as any
+      if (job?.status === 'SERVICE_DRAFT') {
+        payload['SERVICE_FREQUENCY'] = 'EVERY_15_MIN'
+      } else if (job?.status === 'SERVICE_PENDING') {
+        payload['tempExprId'] = 'EVERY_15_MIN'
+        payload['jobId'] = job.id
+      }
+
+      job?.status === 'SERVICE_PENDING' ? this.store.dispatch('job/updateJob', payload) : this.store.dispatch('job/scheduleService', payload);
+    }
+  },
+  setup() {
+    const customPopoverOptions: any = {
+      header: 'Schedule inventory hard sync',
```

Comment:
I think this needs to be internationalised

Reply:
Added text for internationalization.